spec/redirect.test.sh

```diff
@@ -277,3 +277,14 @@ echo second 1>&8
 echo CONTENTS
 cat $TMP/rw.txt
 # stdout-json: "line=first\nCONTENTS\nfirst\nsecond\n"
+
+### &>> appends stdout and stderr
+echo "ok" > $TMP/f.txt
+stdout_stderr.py &>> $TMP/f.txt
+grep ok $TMP/f.txt >/dev/null && echo 'ok'
+grep STDOUT $TMP/f.txt >/dev/null && echo 'ok'
+grep STDERR $TMP/f.txt >/dev/null && echo 'ok'
+# stdout-json: "ok\nok\nok\n"
```

Comment:
Can you change this to use the new style?  Example here:

https://github.com/oilshell/oil/commit/9a9d3421f06bd801dcee8e03af3984c692dc65ba#diff-403617b9ff0265fda249640593bda3c1

The `stdout-json` is hard to read when there are multiple lines.

Reply:
Sure, I can make that change.